domain/machine/state/types.go

```diff
@@ -107,14 +107,19 @@ type machineStatus struct {
 	Updated sql.NullTime `db:"updated_at"`
 }
 
-// setMachineStatus represents the struct to be used for the columns of the
-// machine_status table within the sqlair statements in the machine domain.
+type setStatusInfo struct {
+	StatusID int        `db:"status_id"`
+	Message  string     `db:"message"`
+	Data     []byte     `db:"data"`
+	Updated  *time.Time `db:"updated_at"`
+}
+
 type setMachineStatus struct {
-	MachineUUID machine.UUID `db:"machine_uuid"`
 	StatusID    int          `db:"status_id"`
 	Message     string       `db:"message"`
 	Data        []byte       `db:"data"`
 	Updated     *time.Time   `db:"updated_at"`
+	MachineUUID machine.UUID `db:"machine_uuid"`
```

Comment:
Based on @manadart s recent email, a string would be preferred here.

Reply:
It's not something I'm going to change right now. This will be done in once change for machine.